docs: translate blogs on 08/2021 (#680)
iresty/iresty.github.io: website/blog/2021/08/21/shanghai-meetup.md

---
title: August 21 Apache APISIX Meetup Shanghai, welcome to register!
keywords:
- API Gateway
- APISIX
- Apache APISIX
- Meetup
description: This Meetup was initiated by api7.ai https://www.apiseven.com/, and invited partners such as Aiki and Air CloudHub to present it. Apache APISIX PMC members, contributors, and community technical experts gathered to discuss Apache APISIX community development, industry practices, and other topics. 
tags: [Events]
---

> > This Meetup is initiated by [api7.ai](https://www.apiseven.com/), and invited partners such as Aiki and Air CloudHub to present it. Apache APISIX PMC members, contributors, and community technical experts gathered to discuss Apache APISIX community development, industry practices, and other topics.

Reviewer: ```suggestion
> This Meetup is initiated by [api7.ai](https://www.apiseven.com/), and invited partners such as IQiYi and Air CloudHub to present it. Apache APISIX PMC members, contributors, and community technical experts gathered to discuss Apache APISIX community development, industry practices, and other topics.
```

> <! --truncate-->

Reviewer: ```suggestion
<!--truncate-->
```

> <! --truncate-->

Reviewer: ```suggestion
<!--truncate-->
```

#### Time: August 21, 2021, 13:30

#### Location: Jinghe Island II, 6th Floor, Central Plaza, Huangpu District, Shanghai

#### By attending this Meetup, you can

- Discuss the past and present of the Apache APISIX community

> - Hear from Akiyo scientists about how Akiyo uses Apache APISIX to support a service with 10 million calls

Reviewer: ```suggestion
- Hear from iQiYi scientists about how iQiYi uses Apache APISIX to support a service with 10 million calls
```

> - How AirCloudHub uses Apache APISIX to balance user experience, clean architecture and data sovereignty

Reviewer: ```suggestion
- How AirWallex uses Apache APISIX to balance user experience, clean architecture and data sovereignty
```

- Learn about the benefits of centralized authentication and how to do it on APISIX

If you're interested, don't miss it, **scan the event poster for QR code to register!**

Tips: We will control the size of the event as the epidemic is not relaxed, so registration is limited.

## Event Schedule

> The event schedule![Apache APISIX Meetup](/img/blog_img/2021-08-21-1.jpg)

Reviewer: ```suggestion
![Apache APISIX Meetup](/img/blog_img/2021-08-21-1.jpg)
```

> The event schedule! [Apache APISIX Meetup](/img/blog_img/2021-08-21-1.jpg)

Reviewer: ```suggestion
![Apache APISIX Meetup](/img/blog_img/2021-08-21-1.jpg)
```

## Introduction to the topic

### Apache APISIX based, Aiki API gateway implementation practice

#### Sharing Guests

> Wenjie Jiang / Scientist, Aiki

Reviewer: ```suggestion
Wenjie Jiang / Scientist, iQiYi
```

#### Topic Details

API gateway has become an indispensable part of microservice architecture. It is the only portal for services to the outside world, but also needs to meet the common functions of many applications. As an online video company, Aqiyi needs to carry tens of millions of calls every day and has extremely strict requirements for data security, user request response time, and system stability, so the company needs to build a high-performance and highly available API gateway. This article briefly introduces the architecture of Aiki API gateway and the functions implemented, including request distribution, conditional routing, API management, flow-limiting fusion, security triple prevention, monitoring and alarming, and full-link tracking.

### How to use Apache APISIX for centralized authentication

#### Sharing Guests

Xinxin Zhu / api7.ai Engineer, Apache APISIX Committer

#### Topic Details

Authentication is a very important part of zero-trust architecture. APISIX is a dynamic, real-time, high-performance API gateway that supports a rich set of authentication plugins.

### Data Sovereignty and Apache APISIX Gateway Landing Practice

#### Sharing Guests

> Yang Li / Head of AirCloudHub Technology Platform

Reviewer: ```suggestion
Yang Li / Head of AirWallex Technology Platform
```

#### Topic Details

Data is everywhere, and massive amounts of data are being processed, transmitted, and stored all over the world every moment. However, the world of data is not a place outside the law, and the collection and processing of data must comply with the laws of each country. The API gateway has a special position in the enterprise architecture and therefore plays a special role in the data sovereignty scenario.

### The One Who Walks Alone Goes Fast, The Many Who Walk Far, Apache APISIX Thanks to You

#### Sharing Guests

Yansheng Wang / Co-founder & CTO of api7.ai

#### Topic Details

Apache APISIX has been growing as a community since the first day of open source. In just two years, the product has won the first place in many technical indicators and quickly become the most active API gateway project in the world, and has defined the path that belongs to Apache APISIX: unified 7-layer traffic processing. One person can go fast, but a group of people can go farther, and we are grateful to you for the growth of Apache APISIX.

## Peripheral Benefits

> Join the Meetup, interact with the instructors, and get a chance to win the latest Apache APISIX peripherals: oversized mouse pads, delicate mugs, Apache APISIX badges, and custom umbrellas.

Reviewer: ```suggestion
Join the Meetup, interact with the instructors, and get a chance to win the latest Apache APISIX gifts: mouse pads, coffee mugs, Apache APISIX badges, and umbrellas.
```

> ! [Apache APISIX Gift](/img/blog_img/2021-08-21-4.jpg)

Reviewer: ```suggestion
![Apache APISIX Gift](/img/blog_img/2021-08-21-4.jpg)
```

## How to participate

### On-site registration

Scan the event poster QR code above to sign up!

### Online Live Streaming

If you can't make it to the Meetup, follow the Apache APISIX video to watch the live stream.

> Live![Apache APISIX Live](/img/blog_img/2021-08-21-2.png)

Reviewer: ```suggestion
![Apache APISIX Live](/img/blog_img/2021-08-21-2.png)
```

> Live! [Apache APISIX Live](/img/blog_img/2021-08-21-2.png)

Reviewer: ```suggestion
![Apache APISIX Live](/img/blog_img/2021-08-21-2.png)
```

## Join the group to communicate

Follow the Apache APISIX public page and reply with the keyword "Shanghai" in the background to join the Apache APISIX Shanghai group.

> ! [Apache APISIX wechat](/img/blog_img/2021-08-21-3.png)

Reviewer: ```suggestion
![Apache APISIX wechat](/img/blog_img/2021-08-21-3.png)
```
